turbopack/crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -643,10 +643,11 @@ pub(crate) async fn analyse_ecmascript_module_internal(
                         Some(ModulePart::evaluation())
                     }
                     ImportedSymbol::Symbol(name) => Some(ModulePart::export((&**name).into())),
-                    ImportedSymbol::Part(part_id) => {
+                    ImportedSymbol::PartEvaluation(part_id) => {
                         evaluation_references.push(i);
-                        Some(ModulePart::internal(*part_id))
+                        Some(ModulePart::internal(*part_id, true))
```

Comment:
```suggestion
                        Some(ModulePart::internal(*part_id, false))
```

`PartEvaluation` and `Part` to create the same module fragment. We don't want to have a separate fragment for eval